Eatery/Eat Now/Detail View Controller/EateriesGridViewController.swift

```diff
@@ -156,9 +156,22 @@ class EateriesGridViewController: UIViewController, MenuButtonsDelegate {
         loadData(false, completion: nil)
     }
     
+    func findEmojis(searchBarText: String) -> String{
+        let emojiDictionary: [String: String] = ["🍔": "burger", "🍟": "fries", "🍕": "pizza", "🐔": "chicken", "🌮": "taco", "🌯": "burrito", "🍳" : "egg", "🍚" : "rice", "🍝" :  "spaghetti", "💩": "nasties"]
+        var modifiedSearchBarText = searchBarText
+        for (emoji, searchText) in emojiDictionary {
+            if searchBarText.containsString(emoji){
+                modifiedSearchBarText = modifiedSearchBarText.stringByReplacingOccurrencesOfString(emoji, withString: searchText + " ")
+            }
+        }
+        
+        return modifiedSearchBarText
+    }
+    
     func processEateries() {
         var desiredEateries: [Eatery] = []
-        let searchQuery = searchBar.text ?? ""
+        let searchBarText = findEmojis(searchBar.text ?? "") //replace unicode emoji w/ text name
+        let searchQuery = searchBarText
```

Comment:
You could just combine this line with the previous line, so let searchQuery = findEmojis(searchBar.text ?? "").